sklearn/inspection/tests/test_permutation_importance.py

```diff
@@ -23,8 +23,21 @@
 from sklearn.preprocessing import scale
 from sklearn.utils import parallel_backend
 from sklearn.utils._testing import _convert_container
+from sklearn.utils.estimator_checks import _NotAnArray
 
 
+def test_array_function_not_called():
```

Comment:
To provide context for future maintainers that are not familiar with NEP18:

```suggestion
def test_array_function_not_called():
    """Check that `__array_function__` (NEP18) is not called."""
```

Small nit: may this test be moved to the end of the file?